rali/rali/source/rali_api_augmentation.cpp

```diff
@@ -1543,7 +1567,10 @@ raliCrop(
         output_info.height(input->info().height_single());
         output = context->master_graph->create_image(output_info, is_output);
         output->reset_image_roi();
-        context->master_graph->add_node<CropNode>({input}, {output})->init(crop_h, crop_w, x_drift, y_drift);
+        std::shared_ptr<CropNode> crop_node =  context->master_graph->add_node<CropNode>({input}, {output});
```

Comment:
Codacy found an issue: [Either the condition '!context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)